src/main/java/frc/robot/commands/DropNoteCommand.java

```diff
@@ -5,17 +5,17 @@
 import frc.robot.constants.ScoringConstants;
 import frc.robot.subsystems.Scoring;
 
-public class ScoreNoteCommand extends Command {
+public class DropNoteCommand extends Command {
```

Comment:
Not sure I understand what this command is supposed to do. DropNote sounds like we are trying to get it out of the robot without scoring it. Is that the case? if not then maybe ScoreNoreAmpCommand?